app/gui/src/project-view/util/callTree.ts

```diff
@@ -14,6 +14,9 @@ import type { AstId } from '@/util/ast/abstract'
 import { findLastIndex, tryGetIndex } from '@/util/data/array'
 import type { ExternalId } from 'ydoc-shared/yjsModel'
 import { assert } from './assert'
+// For documentation link.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { GraphDb } from '@/stores/graph/graphDatabase'
```

Comment:
I'm not sure it's necessary. I think we have many places with invalid docs link, but we don't generate documentation as well.

Reply:
Removed